### ISSUE TITLE
How to customize the log format?

### ISSUE THREAD

**Issue by iwilsonlee** [pinojs/hapi-pino#110]
Hi guys,
How to customize the log format in this plugin? Now the log time format is the timestamp, and I want to convert it to the customize formatter, who can tell me how to do this?
thanks for the help.

**Comment by kungfutse** [comment]
I haven't tried, but looking at the documentation, you might be able to do it using pino's log formatting: https://github.com/pinojs/pino/blob/HEAD/docs/api.md#log. You would have to create the pino instance yourself and pass it to hapi-pino as instance as I don't think that hapi pino supports customizing the formatters in the plugin registration options. This would be a nice feature though

**Comment by VirgileD** [comment]
Yes, in fact you can, just put the pino option you need in the hapi-pino options. Hapi-pino options are given to pino as options there: https://github.com/pinojs/hapi-pino/blob/44cbdec76cb29df2d8e90a4efd35a206b7b02133/index.js#L51
